model/src/main/proto/exploration.proto

```diff
@@ -421,31 +421,37 @@ enum CheckpointState {
   CHECKPOINT_UNSAVED = 3;
 }
 
-// Corresponds to a item selection answer that user has selected.
+// Represents the state of item selection answers that the user has selected.
 message ItemSelectionAnswerState {
+  // List of selected item indexes
```

Comment:
Nits:
```suggestion
  // List of selected item indices.
```

I think indices is the correct plural of index.

Reply:
Both are plural, but indices commonly used in technical contexts. Changed.